### ISSUE TITLE
[반도체 박막두께 분석 대회] 새로 접한 NN 스킬

### ISSUE THREAD

**Issue by inh2613** [Sejong-Kaggle-Challengers/JeongInhee#4]
하나하나 중요한 것들이라 정리하고 실습이 필요할 것 같다.
일단 정리만 해본다면 
1. Learning rate 스케줄링
2. Batch normalization , Layer normalization
3. Skip connection
4. 최적화 할 때, Adam에서 발전된 AdamW

을 새로 접했다
